src/main/java/Task.java

```diff
@@ -0,0 +1,21 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
```

Comment:
Good job on have white space between public class and your initialisation of variables